content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -114,6 +114,16 @@ kubectl api-resources --namespaced=true
 kubectl api-resources --namespaced=false
 ```
 
+## Automatic labelling
+
+{{< feature-state state="beta" for_k8s_version="1.21" >}}
+
+The Kubernetes control plane adds an immutable {{< glossary_tooltip text="label(s)" term_id="label" >}}
```

Comment:
should the tooltip text here be in singular, as this is a single label?

Reply:
Good spot, thanks.